rsdroid/build.gradle

```
@@ -103,13 +104,18 @@ task generateTranslations(type: Exec) {
 
 preBuild.dependsOn "generateTranslations"
 
-Boolean wantAllPlatforms = System.getenv("CURRENT_ONLY") != "true"
+Boolean wantAllPlatforms = System.getenv("ALL_ARCHS") == "1"
+print("archs: ${System.getenv("ALL_ARCHS")}")
```

A little less terse message?

```suggestion
print("building all archs? ${System.getenv("ALL_ARCHS")}")
```